tracim/tracim
# Fix/4224  integration fixes for advanced search

#4224 

## Checkpoints



**For developers**

  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**


**For testers**



To test:
- Verify that a file name has its extention at:
    - Recent Activities (dashboard)
    - Search (advanced)
    - Search (simple)
    - Content list ("all contents")
    - App file
- Verify if result's breadcrumbs at an advanced search:
    - doesn't disappear when opening the filters
    - works at all screen types (overflow)
    - has the space name
- Last modification type is translated at advanced search and activity feed
- The information column in advanced search has aligned elements (for all types)
